tests/jobs_fixtures/app33-evm_connection_edb_permissions_and_roles_layer/permissions.sol

```diff
@@ -9,62 +10,62 @@ pragma solidity >=0.4.0;
 * @dev Permissions permissions = Permissions(address(keccak256("Permissions")));
 */
 contract Permissions {
-  /**
-  * @notice Adds a role to an account
-  * @param _account account address
```

Comment:
We had probably better put these back on reflection... It appears to be the style used by product team. It's to distinguish globals from returns. Shadowing is particularly bad in solidity because you tend to declare your returns outside of the body so it may be a little harder to notice.